tests/test_client.sh

```
@@ -46,6 +46,14 @@ dest=$tmpdir/gitlab-testing
 repo="https://gitlab.com/nate-d-olson/snaketestworkflow"
 runTest 0 $output snakedeploy deploy-workflow "${repo}" "${dest}" --name snake-test --branch master
 
+echo
+echo "#### Testing snakedeploy deployment pinned to a specific commit"
+dest=$tmpdir/commit-testing
+repo="https://github.com/snakemake-workflows/dna-seq-varlociraptor"
+commit_sha="87709354b54391aee5dbb01a64cacfc20aed5ec3"
+runTest 0 $output snakedeploy deploy-workflow "${repo}" "${dest}" --branch master --commit ${commit_sha} --name dna-seq-commit
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)